lms/djangoapps/courseware/views/index.py

```diff
@@ -618,14 +622,3 @@ def save_positions_recursively_up(user, request, field_data_cache, xmodule, cour
             save_child_position(parent, current_module.location.block_id)
 
         current_module = parent
-
-
-def show_courseware_mfe_link(user, staff_access, course_key):
-    """
-    Return whether to display the button to switch to the Courseware MFE.
-    """
-    return courseware_mfe_is_visible(
-        course_key=course_key,
-        is_global_staff=user.is_staff,
-        is_course_staff=staff_access,
-    )
```

Comment:
I chose to move this to toggles.py and rename it to `courseware_mfe_is_advertised`, since its new logic reaches into the Waffle Flag values and is very similar to `courseware_mfe_is_visible`. Having the two functions defined side-by-side should make it easier to spot the subtle difference between them.

Reply:
Nit: Again, my preference is that they both call common logic - but not a must.